webrender/src/prim_store.rs

```diff
@@ -290,6 +291,7 @@ pub struct TextRunPrimitiveCpu {
     pub color: ColorF,
     pub render_mode: FontRenderMode,
     pub resource_address: GpuStoreAddress,
+    pub stackingcontext_index: usize,
```

Comment:
I think there's probably a cleaner way to achieve this without requiring storing the stacking context index here. I'll take a closer look at this tomorrow.